src/controller/python/chip/internal/CommissionerImpl.cpp

```diff
@@ -111,8 +112,12 @@ extern "C" chip::Controller::DeviceCommissioner * pychip_internal_Commissioner_N
         chip::Crypto::P256Keypair ephemeralKey;
 
         // Initialize device attestation verifier
-        // TODO: Replace testingRootStore with a AttestationTrustStore that has the necessary official PAA roots available
+#ifdef MATTER_FILE_SYSTEM_ENABLED
+        const chip::Credentials::AttestationTrustStore * testingRootStore =
+            chip::Credentials::GetTestFileAttestationTrustStore("development_trust_store.paa");
```

Comment:
Suggest instead to have the FileAttestationTrustStore be instantiated here, by either making it static here and initializing it every time you come here

Same for chip-tool